packages/engine-test-utils/src/__tests__/engine-server/markdown/backlinksPanelHover.spec.ts

```diff
@@ -0,0 +1,690 @@
+import { ProcFlavor } from "@dendronhq/common-all";
+import { AssertUtils, TestPresetEntryV4 } from "@dendronhq/common-test-utils";
+import {
+  BacklinkOpts,
+  DendronASTDest,
+  MDUtilsV5,
+} from "@dendronhq/engine-server";
+import { runEngineTestV5 } from "../../../engine";
+import { ENGINE_HOOKS } from "../../../presets";
+import { createProcTests, ProcTests } from "./utils";
+
+/**
+ * Helper function to run a single test for Backlinks Panel Hover rendering
+ * @param param0
+ * @returns
+ */
+function singleBacklinksPanelTest({
+  testName,
+  noteBody,
+  backlinkHoverOpts,
+  match,
+}: {
+  testName: string;
+  noteBody: string;
+  backlinkHoverOpts: BacklinkOpts;
+  match: string[];
+}) {
+  return createProcTests({
+    name: testName,
```

Comment:
ah, this is deprecated. can you mark this as such and use `createProcCompileTests` instead? 

see sample usage here > https://github.com/dendronhq/dendron/blob/enhance/backlinks-panel-hover/packages/engine-test-utils/src/__tests__/engine-server/markdown/v5/hierarchies.spec.ts#L7:L7

Reply:
Sounds good.  Next time, can we more proactively tag old methods with `@deprecated`?